bootstrap/README.md

```diff
@@ -11,17 +11,22 @@ comments to the Moderator database.
 
 To generate a JWT token for a service user:
 
-* Set the `TOKEN_SECRET` and `TOKEN_ISSUER` environment variables to the value
-they are set to in the instance of Moderator you are using. They will either be
-set as environment variables in the deployment process, or set to the default values
-in the Moderator [config](../packages/config/index.js).
-* Run the following command from the `/packages/cli` directory.
+* Set up the `TOKEN_SECRET` (and `TOKEN_ISSUER`) configuration items as described
+in the [root README](../README.md)
 
+* Create a service user (if you haven't already got one) and remember his USER_ID:
```

Comment:
Maybe: 
s/remember his USER_ID/remember his USER_ID ("Robot" below): /

Reply:
Actually, the User ID is an integer.  It gets printed out when you create the user.  I'll make this clearer in the text.  (Eventually I hope to replace this section entirely when we can create service users via the UI.)